object_manager_test.go

```diff
@@ -349,6 +351,151 @@ var _ = Describe("Object Manager", func() {
 		})
 	})
 
+	Describe("Create IPv6 Network", func() {
+		cmpType := "Docker"
+		tenantID := "01234567890abcdef01234567890abcdef"
+		netviewName := "default_view"
+		cidr := "2001:db8:abcd:0012::0/64"
+		networkName := "private-net"
+		fakeRefReturn := "ipv6network/ZG5zLm5ldHdvcmskODkuMC4wLjAvMjQvMjU:2001:db8:abcd:0012::0/64/default_view"
+		nwFakeConnector := &fakeConnector{
+			createObjectObj: NewIPv6Network(netviewName, cidr, nil),
+			resultObject:    NewIPv6Network(netviewName, cidr, nil),
+			fakeRefReturn:   fakeRefReturn,
+		}
+		nwFakeConnector.resultObject.(*IPv6Network).Ref = fakeRefReturn
+		objMgr := NewObjectManager(nwFakeConnector, cmpType, tenantID)
+
+		nwFakeConnector.createObjectObj.(*IPv6Network).Ea = objMgr.getBasicEA(true)
+		nwFakeConnector.createObjectObj.(*IPv6Network).Ea["Network Name"] = networkName
+
+		nwFakeConnector.resultObject.(*IPv6Network).Ea = objMgr.getBasicEA(true)
+		nwFakeConnector.resultObject.(*IPv6Network).Ea["Network Name"] = networkName
+		var actualNetwork *IPv6Network
+		var err error
+		It("should pass expected IPv6 Network Object to CreateObject", func() {
+			actualNetwork, err = objMgr.CreateIPv6Network(netviewName, cidr, networkName)
+		})
+		It("should return expected IPv6 Network Object", func() {
+			Expect(actualNetwork).To(Equal(nwFakeConnector.resultObject))
+			Expect(err).To(BeNil())
+		})
+	})
+
+	Describe("Allocate IPv6 Network", func() {
+		cmpType := "Docker"
+		tenantID := "01234567890abcdef01234567890abcdef"
+		netviewName := "default_view"
+		cidr := "2001:db8:abcd:0012::0/64"
+		prefixLen := uint(65)
+		networkName := "private-net"
+		cidr1 := fmt.Sprintf("func:nextavailablenetwork:%s,%s,%d", cidr, netviewName, prefixLen)
+		fakeRefReturn := fmt.Sprintf("ipv6network/ZG5zLm5ldHdvcmskODkuMC4wLjAvMjQvMjU:%s/%s", cidr, netviewName)
+		anFakeConnector := &fakeConnector{
+			createObjectObj: NewIPv6Network(netviewName, cidr1, nil),
+			resultObject:    BuildIPv6NetworkFromRef(fakeRefReturn),
```

Comment:
We are going to allocate network with prefix length 65 but here I see 'fakeRefReturn' which is based on 'cidr' with prefix length 64. Could you explain, why?